TestDataAccess/JSONReader.cs

```diff
@@ -9,72 +9,63 @@ namespace TestDataAccess
 {
     public class JSONReader
     {
-        public static Dictionary<string, string[]> TestValues;
-        public static Dictionary<string, string[]> TestJsonArrayValues;
-        public static Dictionary<string, string> TestCaseValues;
-        public static int ListCount;
+        private JSONFile JsonFile;
```

Comment:
Store the JsonFile as a property of the JsonReader, instead of referencing in each constructor.